playbooks/roles/mysql/files/edxapp.sql

```diff
@@ -2,4 +2,5 @@ UPDATE auth_user SET is_active=False where username="staff";
 UPDATE auth_user SET is_active=False where username="honor";
 UPDATE auth_user SET is_active=False where username="audit";
 UPDATE auth_user SET is_active=False where username="verified";
+ALTER TABLE auth_user ADD UNIQUE (email);
```

Comment:
Would remove the comment about it being idempotent, since all of this automation should be.